### PR TITLE
LPS-190132 [EVP] Update Request Status When Organization Is Review

### PR DESCRIPTION
**Pull request by Elvinho**

### CODE REVIEW

**Comment by eliasantosf** [comment]
Just starting reviewing :)

**Comment by eliasantosf** [comment]
Just starting reviewing :)

**Comment by Elvinho** [comment]
ci:test:sf

**Review by eliasantosf — APPROVED** [review]
LGTM :fire: Thanks @Elvinho

**Comment by Elvinho** [comment]
ci:test:sf

File: workspaces/liferay-evp-workspaces/client-extensions/liferay-evp-etc-spring-boot/src/main/java/com/liferay/evp/BaseRestController.java [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+import org.springframework.web.reactive.function.client.WebClient;
+
+import reactor.util.retry.Retry;
+
+/**
+ * @author Elvison Victor
+ */
+public abstract class BaseRestController {
+
+	protected JSONObject get(Consumer<String> function, Jwt jwt, String path) {
```
Comment (nNilton) [inline workspaces/liferay-evp-workspaces/client-extensions/liferay-evp-etc-spring-boot/src/main/java/com/liferay/evp/BaseRestController.java:30]: please rename function to consumer

**Review by nNilton — CHANGES_REQUESTED** [review]

**Comment by Elvinho** [comment]
ci:test:sf

**Review by nNilton — APPROVED** [review]

**Comment by Elvinho** [comment]
ci:test:sf

**Comment by silasbsi** [comment]
ci:test:relevant

**Comment by Elvinho** [comment]
ci:test:stable

**Comment by Elvinho** [comment]
ci:test:stable

**Comment by Elvinho** [comment]
ci:test:relevant

**Comment by silasbsi** [comment]
ci:forward

**Pull request closed without merging**